pkg/client/userd/grpc.go

```diff
@@ -617,6 +685,13 @@ func (s *Service) RunCommand(cmdStream rpc.Connector_RunCommandServer) (err erro
 			return
 		}
 
+		var rd io.Reader
+		if ctx, rd, cmdErr = stdinPump(ctx, cmdStream, needsPTY(cmd)); cmdErr != nil {
```

Comment:
Do we want to log this error or make it known to the user somehow?

Reply:
It gets known to the user. It's assigned to `cmdErr` which is handled in the defer function that sends the final error.